core/types/block_ext_test.go

```diff
@@ -106,3 +111,410 @@ func assertDifferentPointers[T any](t *testing.T, a *T, b any) {
 	// Note: no need to check `b` is of the same type as `a`, otherwise
 	// the memory address would be different as well.
 }
+
+func TestBlockExtraGetWith(t *testing.T) {
```

Comment:
nit: not sure I understand what "GetWith" means here?

Reply:
Test should be removed indeed (Arr4n is on it)

Reply:
this is removed